src/test/java/org/svydovets/util/EntityReflectionUtilsTest.java

```diff
@@ -71,4 +82,98 @@ public void shouldReturnEntityFieldsWithoutIdSortedByName() throws Exception {
         Assertions.assertThat(EntityReflectionUtils.getUpdatableFields(personClass))
                 .isEqualTo(expected);
     }
+
+    @Test
+    public void shouldReturnEntityFieldWithOneToOneAnnotation() {
+        var anyField = Arrays.stream(Address.class.getDeclaredFields())
+                .filter(field -> field.isAnnotationPresent(OneToOne.class))
+                .findAny()
+                .orElseThrow();
+
+        assertTrue(EntityReflectionUtils.isEntityField(anyField));
+    }
+
+    @Test
+    public void shouldReturnEntityFieldWithManyToOneAnnotation() {
+        var anyField = Arrays.stream(Note.class.getDeclaredFields())
+                .filter(field -> field.isAnnotationPresent(ManyToOne.class))
+                .findAny()
+                .orElseThrow();
+
+        assertTrue(EntityReflectionUtils.isEntityField(anyField));
+    }
+
+    @Test
+    public void shouldReturnThrowForEntityFieldWithManyToOneAnnotationAndExcludeJoinColumnAnnotation() {
```

Comment:
Something wrong with method name, "shouldReturnThrow" ? :)
I guess it's must be "shouldThrowAnnotationMappingExceptionForEntityFieldWithManyToOneAnnotationAndExcludeJoinColumnAnnotation"